lib/travis/api/app/schedulers/schedule_cron_jobs.rb

```diff
@@ -16,6 +16,9 @@ def self.run
           rescue Travis::Lock::Redis::LockError => e
             Travis.logger.error e.message
           end
+          Travis::Lock.exclusive("enqueue_cron_jobs", postgresql_options) do
+            Metriks.timer("api.v3.cron_scheduler.enqueue").time { enqueue }
```

Comment:
@Tahsin-travis-ci, are you enqueuing cron jobs twice? or am I missing something?